### DISCUSSION TITLE
What is the calculation principle of the service_sla metrics?

### DISCUSSION THREAD

**Discussion by gaiwey** [apache/skywalking#9004] [Q&A] [answered]
skywalking version is 8.9.1.

My skywalking alarm always alerts: Service success rate < 80% in the last 10 minutes. It is normal to check this application, but there is no business request.
I see service_sla = from(Service.*).percent(status == true); for service_sla metric。
But I still don't know how to calculate the metrics value.

**Comment by wu-sheng** [comment, marked as answer]
sla == sucess rate == `status == true` requests / all requests

If there is really no traffic, I don't think we have SLA value, no value doesn't mean `0`. So, I think your description is not accurate, there is occasional traffic there but fails.

**Reply by gaiwey** [reply]
I understand the calculation rules of sla: status == true requests / all requests, thank you.

My description is really wrong, our application has requests, but those requests are all 404, so sla<60% can be explained.

**Reply by misaya98** [reply]
> sla == sucess rate == `status == true` requests / all requests
> 
> If there is really no traffic, I don't think we have SLA value, no value doesn't mean `0`. So, I think your description is not accurate, there is occasional traffic there but fails.

same question ， we found sla is 0%， and at the time  service load is 0.  so is it a feature？
<img width="1672" alt="image" src="https://user-images.githubusercontent.com/28262460/215410937-91750ade-5d73-4ff5-906d-ecde7dc49293.png">
